### PR TITLE
Change from AGPL to Apache

### PR DESCRIPTION
**Pull request by nedbat**
@sarina OK?

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,3 @@
+i18n-tools
+
+Copyright 2013-2014, edX
```
Comment (sarina) [inline NOTICE]: why not 2015

**Comment by sarina** [comment]
you said in standup you have to make a change to setup.py is that true in this repo?

**Comment by nedbat** [comment]
@sarina I bumped NOTICE to 2015.  No need to change setup.py since this one doesn't mention the license (some others do).

**Comment by sarina** [comment]
@nedbat :+1:

**Comment by sarina** [comment]
the last PR had 30 violations and this one has 37. ?

**Comment by nedbat** [comment]
The last build used pylint 1.3.1, this build used pylint 1.4.0, which complains about using .format in log messages.

**Comment by sarina** [comment]
@nedbat ah, I guess we don't pin pylint in this repo?

**Pull request merged**